testing/snapshot-test.md

```diff
@@ -0,0 +1,20 @@
+# Snapshot test
+
+## Introduction
+Snapshot tests are used to test visual regression of presentational components and for reducers in order to avoid deep comparison boilerplate code
```

Comment:
there are also some use cases when they are useful for selectors e.g - tracking payloads which pull in different parts of the state tree.

Reply:
Can we do snapshots on state then over snapshots of the whole component structure?